doc/source/qatar.md

```diff
@@ -0,0 +1,53 @@
+# Qatar
+
+The world gives us the opportunity to discover fantastic destinations where some of them seem to be taken from a fairy tale. In hindsight, Qatar was a sparsely developed and sparsely inhabited desert, but this small part of the world has become a highly visited place for tourism in recent years and this tells us how a good conglomerate of effort can definitely change destiny from a town.
+Qatar is a sovereign Arab state that is located in western Asia and has a single border with Saudi Arabia, since the rest of the territory is bathed by the waters of the Persian Gulf. This country has the third largest natural gas reserve in the world and that is why Qatar has the highest per capita income in the world, and is also the nineteenth most peaceful country on the planet.
```

Comment:
```suggestion
Qatar is a sovereign Arab state that is located in western Asia and has a single border with Saudi Arabia, since the rest of the territory is bathed by the waters of the Persian Gulf. This country has the third largest natural gas reserve in the world and that is why Qatar has the highest per capita income in the world, and is also the nineteenth most peaceful country on the planet.

```